src/variable.cpp

```diff
@@ -77,6 +87,36 @@ void Variable::swap(Variable &rhs)
     std::swap(this->mPimpl, rhs.mPimpl);
 }
 
+void Variable::addEquivalence(const VariablePtr &equivalentVariable)
+{
+    setEquivalentTo(equivalentVariable);
+    VariablePtr *pV = new VariablePtr(this);
```

Comment:
For anyone that happens to be looking through this now, I definitely plan to sort out a safer and more C++11 way to do this tomorrow. I was just using this to check that the rest of the current code passes the variable-side connections tests. 

Of course, any suggestions are always welcome :-)